release-notes.md

```diff
@@ -8,6 +8,7 @@ The following updates were made in this release:
 **Android Core 1.4.2**
 
 * Fixed a bug where the replacement value for token `{%~timestampz%}` might be created with the wrong locale.
+* Added  `getUniqueIdentifier()` to the `Event` object. This value is a globally unique identifier for every `Event`.
```

Comment:
Can you please move he 2nd sentence to start on a new line under the bullet? Other than that, it looks good!